packages/story-editor/src/components/panels/design/textStyle/stylePresets.js

```diff
@@ -125,7 +125,8 @@ function PresetPanel({ pushUpdate }) {
           </MoreButton>
           <Popup
             topOffset={topOffset}
-            isRTL={isRTL}
+            // Temp fix, should work the same way as the new color picker (after ongoing refactor is done).
```

Comment:
Same here, i think this as a new issue and the issue number linked here would be helpful.

Reply:
👍🏻  
Done in 215edae2d8a2a4e939bd2a5f0a62015d59d8c9c2